### ISSUE TITLE
Configure Continuous Delivery

### ISSUE THREAD

**Issue by lefoanem** [NorhanKhaled/SkillsAcademy-IOT#57]
[
![Screenshot from 2020-07-23 13-02-14](https://user-images.githubusercontent.com/43403234/88279930-26dfce00-cce5-11ea-9aab-3beef9d11c8f.png)
![Screenshot from 2020-07-23 13-02-27](https://user-images.githubusercontent.com/43403234/88279953-33fcbd00-cce5-11ea-8a8c-e3e45e99b92f.png)
![Screenshot from 2020-07-23 13-02-35](https://user-images.githubusercontent.com/43403234/88279989-4545c980-cce5-11ea-92d6-850098a3826e.png)

Hello I am struggling to deploy my Node-RED application it gives me an error indicated on the screen shots, Please assist so that I can get on with the other exercises.

**Comment by islam3rafah** [comment]
Are you still facing this issue, sometimes it takes few minutes before the app can be deployed

**Comment by lefoanem** [comment]
> Are you still facing this issue, sometimes it takes few minutes before the app can be deployed

Yes I am still facing the issue, when I check resource list I only find service and app are created for it, nothing under cloud foundry apps
![Screenshot from 2020-07-23 14-28-43](https://user-images.githubusercontent.com/43403234/88286443-0b7ac000-ccf1-11ea-9ae1-c4ef7f99ea4f.png)
